### ISSUE TITLE
Only one keybinding for editor.action.clipboardPasteAction works (macOS)

### ISSUE THREAD

**Issue by AndiDog** [Microsoft/vscode#41669]
- VSCode Version: Code 1.19.2 (490ef761b76b3f3b3832eff7a588aac891e5fe80, 2018-01-10T15:49:04.682Z)
- OS Version: Darwin x64 17.3.0
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
vagrant|bbe|0.5.0
vscode-eslint|dba|1.4.4
tslint|eg2|1.0.24
vscode-rust|kal|0.4.2
vscode-json5|mrm|1.0.0
python|ms-|0.9.1
cpptools|ms-|0.14.5
clang-format|xav|1.6.1

(4 theme extensions excluded)

---

Steps to Reproduce:

1. On MacOS, default pasting shortcut is Cmd+V. Add Ctrl+V as alternative shortcut, i.e. add this to keybindings.json without disabling the default:
```
{
    "key": "ctrl+v",
    "command": "editor.action.clipboardPasteAction",
    "when": "editorTextFocus && !editorReadonly"
},
```
2. Copy some text to the clipboard
3. In any one-line text widgets (it seems), such as the "Find" or "Find in Files" text entries, press Cmd+V. Nothing will happen. Press Ctrl+V and the text will be pasted.

Note how I did the same for "Select all": it's mapped to both Cmd+A and Ctrl+A in my config, and both shortcuts work fine in those text widgets. Also, {Cmd,Ctrl}+{X,C} are working correctly.

Reproduces without extensions: Yes

**Comment by alexandrudima** [comment]
This is a rather complicated Electron behaviour regarding `cmd+c`, `cmd+x` and `cmd+v`.

Native inputs in Electron do not have a default behaviour that `cmd+v` will paste (This is different than on Windows, where a native input will paste on `ctrl+v`).

The reason `cmd+v` works out of the box is because we have a menu entry for `Edit > Paste` with `cmd+v` as its keybinding. When `cmd+v` is pressed, the Electron menu will intercept that key combination (similar to you expanding the `Edit` menu and clicking `Paste`). The Electron menu does not differentiate from clicking on an action or triggering it via a keybinding. We will then blindly execute the action `editor.action.clipboardPasteAction` (regardless of keybinding rules), which pastes the clipboard to any input (editor and native inputs).

When you add an extra keybinding for `editor.action.clipboardPasteAction`, we will consider that to be your preferred keybinding and will "paint" it in the menu. So the menu entry will change from `cmd+v` to `ctrl+v`. Then, when pressing `cmd+v`, Electron's menu will not intercept that key combination and it will reach our keybinding dispatcher.

Our default rule for `cmd+v` is `editorTextFocus && !editorReadonly`, so if you are not in a writable editor, it will not do anything.

---

TL;DR:

add a more generic `cmd+v`:

```
{
    "key": "cmd+v",
    "command": "editor.action.clipboardPasteAction"
}, 
{
    "key": "ctrl+v",
    "command": "editor.action.clipboardPasteAction",
    "when": "editorTextFocus && !editorReadonly"
}
```
